site/_posts/2020-07-20-breached-password-detection.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: blog-post
+title: How to enable breached password detection with FusionAuth
+description: How can you easily prevent your users from using passwords that have been cracked and made available for sale on the web? By enabling breached password detection with FusionAuth."
+author: Dan Moore
+image: blogs/social-sign-in-django/headerimage.png
+category: blog
+tags: feature-breached-password-detection
+excerpt_separator: "<!--more-->"
+---
+
+While there are many ways for people to authenticate with online systems, usernames and passwords are still commonly used credentials. Unfortunately, many passwords have been compromised and made available on the Internet. When combined with the fact that users often reuse passwords across different systems, this means that your application or site may be at risk through no fault of your own.
+
+<!--more-->
+
+## The danger of compromised passwords
+
+Breached passwords allow unauthorized access to your systems. If a user's credentials are known to another party, that party can access your systems acting as that user. Depending on data and functionality available in your systems, the results of unauthorized access could range from worrying to disastrous.
+
+## How to avoid unauthorized access
+
+Here are some options auth systems often provide to help secure accounts:
+
+* force users to choose hard passwords
```

Comment:
Should we standardize on capitalizing the first character of lists (ordered or unordered)?

Reply:
Done, added to the devdocs doc as well.